src/ReactVapor.ts

```diff
@@ -0,0 +1,26 @@
+import { ILastUpdatedState } from './components/lastUpdated/LastUpdatedReducers';
+import { IFacetState } from './components/facets/FacetReducers';
+import { IFilterState } from './components/filterBox/FilterBoxReducers';
+import { IFacet } from './components/facets/Facet';
+import { IPerPageState } from './components/navigation/perPage/NavigationPerPageReducers';
+import { ILoadingState } from './components/loading/LoadingReducers';
+import { IPaginationState } from './components/navigation/pagination/NavigationPaginationReducers';
+
+export interface IReactVaporState {
+  lastUpdatedComposite?: ILastUpdatedState[];
+  facets?: IFacetState[];
+  filters?: IFilterState[];
+  perPageComposite?: IPerPageState[];
+  paginationComposite?: IPaginationState[];
+  loadings?: ILoadingState[];
+}
+
+export interface IReduxActionPayload {
```

Comment:
Oops sorry didn't see this. I think a mix of both would be better. How about `IReactVaporActionsPayload`? It covers all actions' payload, but it still is one payload. That is how I see it at least. I will change it in the next PR (nothing has been published yet, so it won't be a breaking change).